modules/external-load-balancer/variables.tf

```diff
@@ -32,3 +22,13 @@ variable "ssl_policy_self_link" {
   description = "The self link of a compute SSL policy for the SSL certificate."
   type        = string
 }
+
+variable "vpc_address" {
+  description = "The address which will be assigned to the load balancer."
+  type        = string
+}
+
+variable "vpc_application_tcp_port" {
```

Comment:
Most submodule variable names are direct references to other submodule outputs, in this case `module.vpc.application_tcp_port`. That was the direction taken by #58. These variables appear new now only because the firewall submodule and the port submodule are merged in the vpc submodule.

If we want to revisit the naming decisions again then we should do that in a separate pull request as the convention changes should be applied consistently across the repository. It is important to be comfortable with these names though, as the work landing on the f-reroll branch will serving as the template for our other modules.

Reply:
(Not in terms of specific names, per se, but the convention of how things are named.)

Reply:
I super appreciate the context. 🙌Naw, we don't need to change conventions. As I mentioned, these even match the tcp/udp ones for others and knowing the protocol is important. It was just the `application_tcp_port` one that I found confusing because it's less self-describing as to what it wants from the user so an update to the variable description may be helpful here. Totally doesn't need to be a rename.

We can do that in another PR as well if that description is something that's proliferated across the module. It may even be better fleshed out in other descriptions, and just not here. Like I said, totally non-blocking.